### PR TITLE
feat: Deprecate tools DOCS-609

### PR DESCRIPTION
**Pull request by nicklem**
Deprecates some tools.

### :eyes: Live preview
https://docs-609-deprecate-tools--docs-codacy.netlify.app/release-notes/cloud/cloud-2023-10-25-csslint-jshint-fauxpas-tailor-tslint-deprecation/

### CODE REVIEW

**Review by stefanvacareanu7 — APPROVED** [review]
This looks good to me! The only objection is that we will enforce the actual deprecation on 25th of October, so it may be worth changing the date.

**Review by DMarinhoCodacy — APPROVED** [review]
LGTM!

File: docs/release-notes/cloud/cloud-2023-10-24-csslint-jshint-fauxpas-tailor-tslint-deprecation.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Deprecation of CSSLint, JSHint, Faux Pas, Tailor, TSLint October 24, 2023<!-- DOCS-609 update with correct launch date -->
+
+On October 24th 2023<!-- DOCS-609 update with correct launch date --> we deprecated the following tools: **CSSLint**, **Faux Pas**, **JSHint**, **Tailor**, and **TSLint**.
+
+These tools have become deprecated or stopped being updated by their maintainers and started providing a bad experience for Codacy users either by reporting false positives or causing other unexpected issues.
```
Comment (claudiacarpinteiro) [inline docs/release-notes/cloud/cloud-2023-10-24-csslint-jshint-fauxpas-tailor-tslint-deprecation.md:10]: Not sure if it should be "proving to" instead? 🤔 

```suggestion
These tools have become deprecated or stopped being updated by their maintainers and started providing a bad experience to Codacy users either by reporting false positives or causing other unexpected issues.
```

File: docs/release-notes/cloud/cloud-2023-10-24-csslint-jshint-fauxpas-tailor-tslint-deprecation.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Removal of CSSLint and TSLint January 1, 2024
+
+On January 1st, 2024 we'll be removing the following tools from Codacy: **CSSLint** and **TSLint**.
+
+The remaining deprecated tools (Faux Pas, JSHint, and Tailor) will be removed later. You can follow the [Codacy release notes](https://docs.codacy.com/release-notes/) for further updates.
+
+## If you are using one of these tools
+
+To continue analyzing any affected repositories, enable the following tools in your [organization coding standards](../../organizations/using-coding-standards.md) (recommended) or on the [code patterns page](../../repositories-configure/configuring-code-patterns.md) of affected repositories, based on the deprecated tool you are using:
```
Comment (claudiacarpinteiro) [inline docs/release-notes/cloud/cloud-2023-10-24-csslint-jshint-fauxpas-tailor-tslint-deprecation.md:22]: Tweak for clarity.
Feel free to ignore it if you consider this tweak brings no clarity to the sentence. 😅 

```suggestion
To continue analyzing your repositories, enable the replacement tool for the corresponding deprecated tool listed below in your [organization coding standards](../../organizations/using-coding-standards.md) (recommended) or on the [code patterns page](../../repositories-configure/configuring-code-patterns.md) of each affected repositories:
```

File: docs/release-notes/cloud/cloud-2023-10-24-csslint-jshint-fauxpas-tailor-tslint-deprecation.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+| Deprecated tool | Replacement tool |
```
Comment (claudiacarpinteiro) [inline docs/release-notes/cloud/cloud-2023-10-24-csslint-jshint-fauxpas-tailor-tslint-deprecation.md:24]: What's the behaviour for new repositories? Is there anything worth mentioning? (as we did [for Trivy](https://docs.codacy.com/release-notes/cloud/cloud-2023-10-13-bundler-audit-deprecation/#if-you-are-using-bundler-audit))

Comment (nicklem) [reply docs/release-notes/cloud/cloud-2023-10-24-csslint-jshint-fauxpas-tailor-tslint-deprecation.md:24]: @stefanvacareanu7 @DMarinhoCodacy 

Any caveats worth mentioning here? For Trivy, we mentioned that it's on for default on new repos.

Comment (stefanvacareanu7) [reply docs/release-notes/cloud/cloud-2023-10-24-csslint-jshint-fauxpas-tailor-tslint-deprecation.md]: The deprecated tools are disabled by default but can be enabled and will work as before.

The suggested cloud tools are enabled by default. Clang Tidy is a client-side tool that cannot be enabled by default.

**Review by stefanvacareanu7 — APPROVED** [review]

**Pull request merged**